src/pages/List.js

```diff
@@ -31,7 +32,10 @@ const List = () => {
         // data = an Array containing all of the documents in the collection. Each item will contain an id along with the other data contained in the document.
         render={({ isLoading, data }) => {
           if (!isLoading && data.length === 0) {
-            setLocalStorageToken(getLocalStorageToken());
+            // If a list wasn't found create a new token in case the user adds an item. That way a new list will be created
+
+            // Not working. TODO: Maybe make a function that clears the token in storage and generates a new one?
```

Comment:
Maybe this won't be a problem when we set up the welcome prompt for the app.

Reply:
I think the last changes I made to AddItem and List have too much token logic. I probably need to make a new function from tokenContext.js that regenerates a token to use when a matching list isn't found.